docs/labels.md

```diff
@@ -15,7 +15,7 @@ special meaning:
   * [User-approval pending](#user-approval-pending)<!-- endToc -->
 
 
-## Bump dependencies
+## Bump dependencies
```

Comment:
is it a different whitespace?

Reply:
It looks like it is. I just made a copy-paste from the next title